File: Test1_solution/solution_7.py

def odd_count(arr,n):
    count=0
    for i in range(0,n):
        if((arr[i]%2)!=0):
            count+=1
    return count
arr=[2,3,4,5]
print("odd ele in array=",odd_count(arr,len(arr)))

File: Test1_solution/solution_7.java
public class solution_7
{
    public static void main(String args[])
    {
        int arr[]={2,3,5,6};
        System.out.println("arr legnth="+arr.length);
        int result=odd_count(arr,arr.length);
        System.out.println("No of odd ele="+result);
    }

    public static int odd_count(int arr[],int n)
    {
        int count=0;
        for(int i=0;i<n;i++)
        {
            if((arr[i]%2)!=0)
                count++;
        }
        return count;
    }
}